tests/www/views/test_views_tasks.py

```diff
@@ -272,14 +272,30 @@ def test_dag_details_trigger_origin_tree_view(app, admin_client):
         state=State.RUNNING,
     )
 
-    url = 'dag_details?dag_id=test_tree_view'
+    url = 'tree?dag_id=test_tree_view'
     resp = admin_client.get(url, follow_redirects=True)
     params = {'dag_id': 'test_tree_view', 'origin': '/tree?dag_id=test_tree_view'}
     href = f"/trigger?{html.escape(urllib.parse.urlencode(params))}"
     check_content_in_response(href, resp)
 
 
-def test_dag_details_trigger_origin_graph_view(app, admin_client):
+def test_dag_details_trigger_origin_dag_details_view(app, admin_client):
+    app.dag_bag.get_dag('test_tree_view').create_dagrun(
+        run_type=DagRunType.SCHEDULED,
+        execution_date=DEFAULT_DATE,
+        data_interval=(DEFAULT_DATE, DEFAULT_DATE),
+        start_date=timezone.utcnow(),
+        state=State.RUNNING,
+    )
+
+    url = 'dag_details?dag_id=test_tree_view'
+    resp = admin_client.get(url, follow_redirects=True)
+    params = {'dag_id': 'test_tree_view', 'origin': '/dag_details?dag_id=test_tree_view'}
+    href = f"/trigger?{html.escape(urllib.parse.urlencode(params))}"
+    check_content_in_response(href, resp)
+
+
+def test_dag_details_trigger_origin_dag_details_view(app, admin_client):
```

Comment:
Same function name makes flake8 a sad bunny

Reply:
Not just flake8, this would actually make the first test not run.

Reply:
I am a copy and paste genius. Changed the name and made the intent of the seeming duplicate test a little clearer, hopefully.